package.json

```diff
@@ -80,20 +83,28 @@
     "eslint-config-airbnb": "^3.1.0",
     "eslint-plugin-react": "^3.14.0",
     "expect.js": "^0.3.1",
-    "flow-bin": "^0.37.0",
     "foreman": "^1.4.1",
     "husky": "^0.13.3",
     "jest": "^20.0.4",
     "lint-staged": "^3.4.0",
-    "prettier": "^1.5.2",
-    "sinon": "^1.17.2"
+    "prettier": "^1.7",
+    "sinon": "^1.17.2",
+    "typescript": "^2.5.2"
   },
   "jest": {
     "testRegex": "/test/.*.js$",
     "setupTestFrameworkScriptFile": "<rootDir>/test/helper.js",
     "testPathIgnorePatterns": ["/node_modules/", "/test/helper.js", "/test/utils.js", "/test/__mocks__"]
   },
+  "prettier": {
+    "printWidth": 120,
+    "semi": false,
+    "singleQuote": false,
+    "trailingComma": "es5",
+    "bracketSpacing": true
+  },
   "lint-staged": {
-    "*.js": ["prettier --write --trailing-comma es5 --print-width 120 --no-semi", "git add"]
+    "*.json": ["yarn prettier --write", "git add"],
+    "*.js": ["eslint --fix", "prettier --write", "git add"]
```

Comment:
adds JSON, as prettier can now handle that too. Adds the eslint auto fix too, because if we're verifying against it, may as well try do our best.